Source/IO.cpp

```diff
@@ -738,11 +771,15 @@ PeleC::writeBuildInfo(std::ostream& os)
      << std::endl;
 #endif
 
-#ifdef PELEC_SPRAY
-  os << std::setw(35) << std::left << "PELEC_SPRAY " << std::setw(6) << "ON"
+#ifdef PELEC_USE_SPRAY
+  os << std::setw(35) << std::left << "PELEC_USE_SPRAY " << std::setw(6) << "ON"
      << std::endl;
 #else
-  os << std::setw(35) << std::left << "PELEC_SPRAY " << std::setw(6) << "OFF"
+  os << std::setw(35) << std::left << "PELEC_USE_SPRAY " << std::setw(6)
+     << "OFF" << std::endl;
+#endif
+#ifdef PELEC_USE_SOOT
+  os << std::setw(35) << std::left << "PELEC_USE_SOOT " << std::setw(6) << "ON"
```

Comment:
Do we want an `#else` on PELEC_USE_SOOT here? Like you did for the spray

Reply:
Yep.